TrafficCapture/dockerSolution/src/main/docker/elasticsearchWithSearchGuard/Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN sed 's/searchguard/plugins.security/g' $ELASTIC_SEARCH_CONFIG_FILE |  \
 # but maybe not for a demo to show individual steps
 RUN /root/enableTlsConfig.sh $ELASTIC_SEARCH_CONFIG_FILE
 # Alter this config line to either enable(searchguard.disabled: false) or disable(searchguard.disabled: true) HTTP auth
-RUN echo "searchguard.disabled: false" >> $ELASTIC_SEARCH_CONFIG_FILE
+RUN echo -n "searchguard.disabled: false" >> $ELASTIC_SEARCH_CONFIG_FILE  && \
+    echo -n "plugins.security.ssl.http.enabled_protocols: ['TLSv1.2', 'TLSv1.3']" >> $PROXY_TLS_CONFIG_FILE
```

Comment:
I see that this sets the allowable protocols to `TLSv1.2` and `TLSv1.3`, but this dockerfile only looks relevant for the `elasticsearchWithSearchGuard` stack, not the capture proxy in general. Is that the full scope of the concern here?

Reply:
@AndreKurait - are we using this as a base image for the proxy that we deploy?